package.json

```diff
@@ -29,16 +29,13 @@
     "test:unit:global": "mocha test/unit-global/*.test.js",
     "test:unit:mocha": "mocha './app/**/*.test.js'",
     "test:unit:lax": "mocha --config '.mocharc.lax.js' './app/**/*.test.js'",
```

Comment:
If we don't have the strict tests anymore, can we get rid fo this too?

Reply:
Yes we can! 074a885c8